src/installer/managed/Microsoft.NET.HostModel/Bundle/Bundler.cs

```diff
@@ -227,6 +234,19 @@ private FileType InferType(FileSpec fileSpec)
             return FileType.Unknown;
         }
 
+        public static ImmutableArray<byte> BundleHeaderPlaceholder = [
```

Comment:
```suggestion
        internal static ReadOnlySpan<byte> BundleHeaderPlaceholder => [
```
ROS property will make roslyn optimize it better.